portfolio/src/main/webapp/index.html

```diff
@@ -8,7 +8,7 @@
     <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
   </head>
   <body onload=fetchBlobstoreUrlAndShowForm()>
-    <div id="content">
+    <div id="content" style="width: 750px">
```

Comment:
maybe put the styling in the style sheet?

Reply:
Yes. Thank you to point it out! I have fixed it in PR #10 .